pkgs/by-name/an/anytype/tantivy-go/default.nix

```diff
@@ -6,17 +6,17 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "tantivy-go";
-  version = "0.3.1";
+  version = "1.0.1";
```

Comment:
This likewise didn't exist at the time of release of `0.45.3` and is a major update.